browse/static/js/catalyzex.js

```diff
@@ -0,0 +1,59 @@
+  }
+
+  const fetchCatalyzeXCode = async () => {
+    const cxApiUrl = `https://www.catalyzex.com/api/code?src=arxiv&paper_arxiv_id=${arxivId}`;
+
+    let result = {};
+
+    try {
+      result = await $.ajax({ url: cxApiUrl, timeout: 2000, dataType: "json" });
+    } catch (error) {
+      result = {};
+    }
+
+    return result;
+  };
+
+  $output.html('');
+
+  const { count: implementations, cx_url: cxImplementationsUrl } = await fetchCatalyzeXCode()
+
+  $output.append("<h2>CatalyzeX</h2>");
+
+  const submitItHereLink = `<a target="_blank" href="https://www.catalyzex.com/add_code">submit it here</a>`;
+
+  if (implementations) {
+    const codeLink = $(`<a target="_blank"></a>`);
+    codeLink.attr("href", cxImplementationsUrl);
+    codeLink
+      .append(icons.github)
+      .append(`${implementations} code implementation${implementations > 1 ? "s" : ""} found on`)
+      .append(icons.catalyzex)
+      .append("CatalyzeX");
+
+    $output
+      .append(codeLink)
+      .append($(`<p>If you have code to share with the community, please ${submitItHereLink}.</p>`))
+  } else {
+    $output.append(`<p>The authors of ${paperTitle ? `"${paperTitle}"` : 'this paper'} have not publicly listed the code yet. If you have code to share with the community, please ${submitItHereLink}.</p>`)
```

Comment:
It's possible that code is somewhere out there by the authors but not been crawled yet by us, so updating the wording
```suggestion
    $output.append(`<p>No code found just yet. If you have code to share with the community, please ${submitItHereLink}.</p>`)
```

Reply:
Sounds good. Here's what the not-found UI looks like after this change:

![Screen Shot 2023-02-03 at 21 30 56](https://user-images.githubusercontent.com/58868651/216734986-c60ed5c1-2886-4750-a1ba-e626c67556cb.png)